components/schema-migrator/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/schema-migrator/cleaner"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	time.Sleep(20 * time.Second)
```

Comment:
Why is sleep at the start time necessary? Perhaps add a comment explaining why 20 seconds.